libs/core/langchain_core/output_parsers/openai_tools.py

```diff
@@ -47,6 +48,18 @@ def parse_tool_call(
         parsed["id"] = raw_tool_call.get("id")
     return parsed
 
+def make_json_valid(json_string: str) -> str:
+    """
+    This function takes a dictionary-like string and converts it into a valid JSON string.
+    It does this by adding double quotes around the keys.
+    """
+    try:
+        corrected_json_string = re.sub(r'(\b\w+\b):', r'"\1":', json_string)
+        return corrected_json_string
```

Comment:
add corrected_json_string.replace("(", "").replace(")", "") add this to it bcz always getting in str(tuples)